samples/blinky/Cargo.toml

```
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 Linaro LTD
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+# This must be rustapp for now.
+name = "rustapp"
+version = "0.1.0"
+edition = "2021"
+description = "A sample hello world application in Rust"
+license = "Apache-2.0 or MIT"
```

```suggestion
license = "Apache-2.0 OR MIT"
```

See <https://spdx.github.io/spdx-spec/v2-draft/SPDX-license-expressions/#d2-case-sensitivity>